update reputations command - is array 1 (#15145)
demisto/content: Packs/CheckPhish/ReleaseNotes/1_0_7.md

#### Integrations
##### CheckPhish

> - You can now pass an array to the ***url*** command.

Reviewer: ```suggestion
- Added array input support for the ***url*** command.
```
